validator/test/fixtures/sql_maintenance_window/constraints/no_hour/data.yaml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: constraints.gatekeeper.sh/v1alpha1
+kind: GCPSQLMaintenanceWindowConstraintV1
+metadata:
+  name: gcp-sql-maintenance-window-v1
```

Comment:
Same here - that way these 2 constraints do not share the same name and the name tells you how this constraint should behave.

Reply:
done

Reply:
did you push the change? I do not see it here - the name is unchanged